database/tables_creation.sql

```diff
@@ -0,0 +1,170 @@
+-- Table: onboarding_schema.users
+
+-- DROP TABLE IF EXISTS onboarding_schema.users;
+
+CREATE TABLE IF NOT EXISTS onboarding_schema.users
+(
+    id integer NOT NULL,
+    name character varying COLLATE pg_catalog."default" NOT NULL,
+    CONSTRAINT users_pkey PRIMARY KEY (id)
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS onboarding_schema.users
+    OWNER to postgres;
+
+-- Table: onboarding_schema.items
+
+-- DROP TABLE IF EXISTS onboarding_schema.items;
+
+CREATE TABLE IF NOT EXISTS onboarding_schema.items
+(
+    id integer NOT NULL,
+    name character varying COLLATE pg_catalog."default" NOT NULL,
+    CONSTRAINT items_pkey PRIMARY KEY (id)
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS onboarding_schema.items
+    OWNER to postgres;
+
+-- Table: onboarding_schema.comments
+
+-- DROP TABLE IF EXISTS onboarding_schema.comments;
+
+CREATE TABLE IF NOT EXISTS onboarding_schema.comments
+(
+    user_sender integer NOT NULL,
+    user_receiver integer NOT NULL,
+    item_commented integer NOT NULL,
```

Comment:
I believe items commented should be on **favourite** items of **other** users, here we would allow commenting on any item whether its favourite or not and whether its from another user or the commenter user itself

I guess we can add this conditions with contraints, also another possible way is to connect the comments to the "mark_as_favourite" instead of the user_receiver and item_commented, that way its linked to a favourite and the user who marked it as favourite